labonneboite/common/maps/README.md

```diff
@@ -0,0 +1,360 @@
+# Isochronous timing
+
+Content:
+- [Introduction](#introduction)
+- [Settings](#settings)
+- [High Level Flow](#high-level-flow)
+- [Cache](#cache)
+- [Asynchronous tasks](#asynchronous-tasks)
+- [Endpoints](#endpoints)
+
+
+## Introduction
+
+Isochronous timing refers to a sequence of events occurring regularly or at equal time intervals. In our project, it adds the possibility to **search companies by commuting time** instead of distance. For example, a user may ask "Find me offices located at 30 minutes by car from Metz city center".
+
+
+### Search area
+
+Searching by distance returns a **disc-shaped list of points**, as it uses a **radius** from a given location.
+
+![](readme_images/traveltimeplatform_radius.png?raw=true)
+*Generated using [Travel Time Platform](https://www.traveltimeplatform.com/)*
+
+On the contrary, searching by commute time generates several polygon-shaped lists of points. *Why?* Because the unit of measure is no longer a **distance** as the crow flies, which is linear, but **time** required to go from point A to point B. It does not depend on the distance but on many **parameters**: transport mode, infrastructures (Is a firm easily reached by bus? Is there a road around and how fast can one drive on it? etc.), public transport interconnections, ... This "breaks up" space and leads to a fragmented action zone: a set of several polygons.
+
+![](readme_images/traveltimeplatform_isochrone.png?raw=true)
+*Reachable zones using public transport in Paris in 30 minutes from city center.*
+
+
+### APIs and requirements
+
+Our project allows searching by two transport modes: car or public transports. To get isochrone polygons given a certain location, we call two APIs:
+
+- car: [IGN](https://geoservices.ign.fr/documentation/geoservices/isochrones.html)
+- public transports: [Navitia](http://doc.navitia.io/#journeys)
+
+
+Requirements:
+
+- **[Huey](https://huey.readthedocs.io/)**: asynchronous task queue used to cache isochrones and durations.
+- **A Redis server** (_optional_): used as a backend for Huey. Recommended in production. Default to local cache.
```

Comment:
The redis server is also used to cache isochrone results. This is important in production, as it takes a long time to compute a single isochrone.

Reply:
Information added!